### ISSUE TITLE
Embedded `ipywidgets` are not rendered

### ISSUE THREAD

**Issue by seeM** [quarto-dev/quarto-cli#2445]
### Bug description

With Quarto 1.2.108 on macOS 12.5.1 (Intel):

1. Clone this repo: https://github.com/seeM/test-jupyter-widgets
2. Run `quarto render`

    You'll see the following warning here. I'm not sure if its related:

    ```
    WARNING: Malformed Jupyter Output Display Data found:
    WARNING: {"model_id":"7fc2753d918e4db8a2b5d774ece32c7b","version_major":2,"version_minor":0}
    ```
4. Open `_site/index.html`
5. The `ipywidgets.IntSlider` widget will not be rendered, and the browser console will have error `Uncaught (in promise) Error: View state has errors: [object Object]` in `embed-amd.js`

---

If you make the following change to the rendered `_site/index.html`, the widget will render correctly (the `model_id` may differ in your `index.html`):

```diff
<script type="application/vnd.jupyter.widget-view+json">
-[{"model_id":"7fc2753d918e4db8a2b5d774ece32c7b","version_major":2,"version_minor":0}]
+{"model_id":"7fc2753d918e4db8a2b5d774ece32c7b","version_major":2,"version_minor":0}
</script>
```

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by jjallaire** [comment]
Thanks for reporting this and especially for the repro and fix :-)

This was a regression in v1.1. It's fixed here (https://github.com/quarto-dev/quarto-cli/commit/4ec99ae8891eebfc1676da4dc6170b32ec65e667) which is available now in our v1.2.110 pre-release (https://quarto.org/docs/download/prerelease.html). I'm also backporting this fix to v1.1 (updated version of which should be available later today).

**Comment by seeM** [comment]
Thanks for the speedy fix! :) I've confirmed that it works in my test repo with 1.2.117
